gettsim/benefits/benefit_checks/vermoegens_checks.py

```diff
@@ -32,15 +32,18 @@ def _wohngeld_basis_hh_vermögens_check(
     TODO: Need to write numbers to params.
```

Comment:
Do it now?

Reply:
Please not. I am really done for today. Just want that it works and then I am out!

Reply:
Created #207